File: Scala/fizzbuzz.scala
//FIZZBUZZ implementation in SCALA
for (i <- 1 to 100) {
	if (i % 3 == 0 && i % 5 != 0) println("FIZZ")
	else if (i % 5 == 0 && i % 3 != 0) println("BUZZ")
	else if (i % 3 == 0 && i % 5 == 0) println("FIZZBUZZ")
	else println(i)
}

File: Java/fizzbuzz.java
/*	This is an implementation of FIZZBUZZ in Java
*/

public class fizzbuzz {
	public static void main(String[] args) {
		for ( int i = 1; i <= 100; i++) {
			if (i%15 == 0) {
				System.out.print("FIZZBUZZ\n");
			} else if (i%3 == 0) {
				System.out.print("FIZZ\n");
			} else if (i%5 == 0) {
				System.out.print("BUZZ\n");
			} else {
				System.out.print(i + "\n");
			}
		}
	}
}
